azure-resources/Sql/servers/recommendations.yaml

```diff
@@ -17,17 +17,17 @@
     - name: Active Geo Replication
       url: "https://learn.microsoft.com/en-us/azure/azure-sql/database/active-geo-replication-overview"
 
-- description: Auto Failover Groups for apps should include all related databases for cohesion
+- description: Use Auto Failover Groups that can include one or multiple databases, typically used by the same application
   aprlGuid: 943c168a-2ec2-a94c-8015-85732a1b4859
   recommendationTypeId: null
   recommendationControl: Disaster Recovery
   recommendationImpact: High
   recommendationResourceType: Microsoft.Sql/servers
   recommendationMetadataState: Active
   longDescription: |
-    You can use the readable secondary databases to offload read-only query workloads. Autofailover groups involve multiple databases configured on a primary server, supporting replication of all databases in the group to only one secondary server or instance in a different region.
+    Failover Groups are a disaster recovery feature that enables you to configure databases contained on a logical server to replicate to a logical server in another region. Failover groups simplify the management of geo-replicated databases and enable a single endpoint that automatically routes connections to the replicated databases in the event of a failure of the primary logical server.
```

Comment:
Please re-word to bring the longDescription under the 300 character limit, including spaces.

Reply:
Fixed